src/ccl/ccl.component.ts

```diff
@@ -3,20 +3,19 @@
  * App Component
  */
 import { Component, ViewEncapsulation } from '@angular/core';
+import { Sidebar } from './sidebar/sidebar.component';
+import { Charts } from './charts/charts-container.component';
+
 
-/*
- * App Component
- * Top Level Component
- */
 @Component({
   selector: 'ccl',
+  directives: [Sidebar, Charts],
   encapsulation: ViewEncapsulation.None,
   styleUrls: [
-  '../assets/css/main.css',
+    './ccl.style.css',
```

Comment:
The naming section of the angular style guide (https://angular.io/styleguide#!#02-01) suggests that this should be called `ccl.component.css` because its an asset for the ccl component.